Units/parser-clojure.r/clojure-methods.b/input.clj

```diff
@@ -0,0 +1,19 @@
+(ns multimethod.test)
+
+(defmulti test
+  (fn [type] type))
+
+(defmulti documented-multimethod "Documentation"
+  (fn [type] type))
+
+(defmethod test nil
+  [& _]
+  nil)
+
+(defmethod test :test
+  [& _]
+  nil)
+
+(defmethod test :test2 named-method
```

Comment:
Could you explain this line?
I wonder why `named-method` should be extracted because my knowledge of Clojure is limited.

Reply:
It's a method-specific name that's added to multimethod function name. See https://clojuredocs.org/clojure.core/defmethod#example-542692c7c026201cdc3269cd

Reply:
No, it doesn't work as a self-sufficient function. It's merely a piece of metadata identifying the method. It's only useful in debugging and (given ctags support) in location search.

Reply:
I see. Thank you. We should introduce a new kind for such a language object. What do Clojure hackers call it? I think of `method-identifier/i` or `method-id/i`.

The ideal tags for `named-method` looks like:
```
named-method	input.clj	/^(defmethod test :test2 named-method$/;"	i	multimethod.test	method:test
test	input.clj	/^(defmethod test :test2 named-method$/;"	m	multimethod.test	identifier:named-method
```

Anyway, the current Clojure parser cannot extract caddr.
So, to satisfy these new test cases, we must rewrite the parser.